src/IIIFPresentation/Core/IIIF/IIIFResponseX.cs

```diff
@@ -1,49 +1,55 @@
-using IIIF;
-using IIIF.Serialisation;
-using Newtonsoft.Json;
-
-namespace Core.IIIF;
-
-public static class IIIFResponseX
-{
-    /// <summary>
-    /// Convert provided stream to Presentation type inheriting from <see cref="JsonLdBase"/>
-    /// </summary>
-    public static async Task<T?> ToPresentation<T>(this Stream contentStream, JsonSerializerSettings? settings = null)
-        where T : JsonLdBase, new()
-    {
-        using var streamReader = new StreamReader(contentStream);
-        return await DeserializeStream<T>(settings, streamReader);
-    }
-    
-    /// <summary>
-    /// Convert provided string to Presentation type inheriting from <see cref="JsonLdBase"/>
-    /// </summary>
-    public static async Task<T?> ToPresentation<T>(this string content, JsonSerializerSettings? settings = null)
-        where T : JsonLdBase, new()
-    {
-        using var streamReader = new StringReader(content);
-        return await DeserializeStream<T>(settings, streamReader);
-    }
-    
-    private static async Task<T?> DeserializeStream<T>(JsonSerializerSettings? settings, TextReader streamReader)
-        where T : JsonLdBase, new()
-    {
-        await using var jsonReader = new JsonTextReader(streamReader);
-
-        settings ??= new(IIIFSerialiserX.DeserializerSettings);
-
-        var serializer = JsonSerializer.Create(settings);
-
-        try
-        {
-            var result = new T();
-            serializer.Populate(jsonReader, result);
-            return result;
-        }
-        catch (JsonReaderException)
-        {
-            return default;
-        }
-    }
+using IIIF;
+using IIIF.Serialisation;
+using Newtonsoft.Json;
+
+namespace Core.IIIF;
+
+public static class IIIFResponseX
+{
+    /// <summary>
+    /// Convert provided stream to Presentation type inheriting from <see cref="JsonLdBase"/>
+    /// </summary>
+    /// <remarks>
+    /// There is a slight difference between the IIIF ToJson method, in that ToPresentation is case-insensitive
+    /// </remarks>
+    public static async Task<T?> ToPresentation<T>(this Stream contentStream, JsonSerializerSettings? settings = null)
+        where T : JsonLdBase, new()
+    {
+        using var streamReader = new StreamReader(contentStream);
+        return await DeserializeStream<T>(settings, streamReader);
+    }
+    
+    /// <summary>
+    /// Convert provided string to Presentation type inheriting from <see cref="JsonLdBase"/>
+    /// </summary>
+    /// <remarks>
+    /// There is a slight difference between the IIIF ToJson method, in that ToPresentation is case-insensitive
+    /// </remarks>
+    public static async Task<T?> ToPresentation<T>(this string content, JsonSerializerSettings? settings = null)
```

Comment:
Could this overload be set up to `inheritdoc` the remarks from the first form of the method? I _think_ there's a way to inherit only specific tags but I'm not 100% sure.

Reply:
it does, but I don't think it works with extension methods yet